openapi/publicapi_spec.json

```diff
@@ -665,7 +665,7 @@
           "name" : "Idempotency-Key",
           "schema" : {
             "type" : "string",
-            "pattern" : "^[a-zA-Z0-9-]+$"
+            "pattern" : "^$|^[a-zA-Z0-9-]+$"
```

Comment:
Is this indicating that the field value can be empty?

Reply:
I forgot to rebase, that was a 🤦 . It was a PR on Friday: https://github.com/alphagov/pay-publicapi/pull/1721

Reply:
I wonder why github actions didn't pick it up 🤔

Reply:
Isn't the expression incorrect for specs?

Reply:
We don't have checks for OpenAPI changes in publicapi